article/app/views/liveblog/liveBlogBody.scala.html

```diff
@@ -50,25 +51,28 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(article.trail.h
 
                     @fragments.witnessCallToAction(article.content)
 
-                    @fragments.liveNotificationsCallToActionPlaceHolder()
-
                     <div class="blog__left-col">
 
                         @fragments.contentMeta(article)
 
                         <div class="js-top-marker"></div>
-                        @if(article.hasKeyEvents) {
-                            <div class="blog__timeline blog__dropdown js-live-blog__key-events modern-visible js-updates-desktop">
-                                <div class="blog__timeline-container js-live-blog__timeline-container" data-component="timeline">
-                                    @fragments.dropdown("Key events", Some("key-events"), true) {
-                                        <ul class="timeline js-live-blog__timeline u-unstyled">
-                                            @views.html.liveblog.keyEvents("/" + article.metadata.id, KeyEventData(article.content.fields.blocks, timezone))
-                                        </ul>
-                                    }
-                                </div>
+                        <div class="js-live-blog__sticky-components">
+                            <div class="js-live-blog__sticky-components-container blog__sticky-components-container">
```

Comment:
I would expect the two classes to be identical except for the `js-` prefix? Though this looks like it's a fallout from previous naming so might be a major change :/